mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/project.py

```diff
@@ -16,8 +18,49 @@ def __init__(self, project_draft: dict) -> None:
 
         # set group size
         self.groupSize = project_draft["groupSize"]
-        self.inputGeometries = project_draft["inputGeometries"]
+        self.geometry = project_draft["geometry"]
```

Comment:
this causes the error in github actions. From the manager dashboard side, do you make sure that this is always set? What is this value for TM ID and AOI?

Reply:
-> setup new test jsons in  tests/integration/fixtures/footprint/project_drafts -> one for each new input type

Reply:
-> execute them in test/integration/test_create_footprint_project.py